runhouse/servers/cluster_servlet.py

```diff
@@ -217,3 +248,141 @@ async def aclear_all_references_to_env_servlet_name(self, env_servlet_name: str)
         for key in deleted_keys:
             self._key_to_env_servlet_name.pop(key)
         return deleted_keys
+
+    ##############################################
+    # Cluster status functions
+    ##############################################
+
+    async def asend_status_info_to_den(self):
+        while True:
+            logger.info("Sending cluster status to Den")
+            interval_size = (await self.aget_cluster_config()).get(
+                "den_status_ping_interval"
+            )
+            if interval_size == STATUS_CHECK_OFF:
```

Comment:
i think we need this to sit inside the `try` block, so that we still execute the `sleep` inside the `finally`. also `break` instead of `return` might be more appropriate here